lib/vdsm/network/netinfo/cache.py

```diff
@@ -289,7 +289,7 @@ def networks_base_info(running_nets, routes=None, ipaddrs=None):
             continue
         iface = get_net_iface_from_config(net, attrs)
         try:
-            if not link_iface.iface(iface).exists():
+            if not vdsm.network.link.iface.iface(iface).exists():
```

Comment:
I am not a huge fan of having the full path now, but it does not hurt anything.

Reply:
Standardized the imports with `from vdsm.network import X` so code will use `X.x.f()`
This will greatly reduce the lookups required while reading the code by clarifying the source of the data and its flow direction.
I will create another PR later to continue with this work.